_specifications/lsp/3.18/language/inlineCompletion.md

```diff
@@ -0,0 +1,233 @@
+#### <a href="#textDocument_inlineCompletion" name="textDocument_inlineCompletion" class="anchor">Inline Completion Request (:leftwards_arrow_with_hook:)</a>
+
+> *Since version 3.18.0*
+
+The inline completion request is sent from the client to the server to compute inline completions for a given text document either explicitly by a user gesture or implicitly when typing.
+
+_Client Capability_:
```

Comment:
This seems to tell very little about what this feature exactly is about.

Is it an alternative to normal completions or does it enhance it?
How does it differ from normal completions?
What problem does it solve that normal completions don't?

Reply:
That are some questions that pop into my head when trying to think about it from the implementation perspective but since I'm not implementing it, I will not create more noise about it here.